RecoLocalTracker/SiPixelRecHits/src/PixelCPEFastParamsHost.cc

```diff
@@ -220,14 +227,29 @@ void PixelCPEFastParamsHost<TrackerTraits>::fillParamsForDevice() {
     // sample x by charge
     int qbin = pixelCPEforDevice::kGenErrorQBins;  // low charge
     int k = 0;
-    for (int qclus = 1000; qclus < 200000; qclus += 1000) {
+    int qClusIncrement = 100;
+    for (int qclus = 1000; k < pixelCPEforDevice::kGenErrorQBins;
+         qclus += qClusIncrement) {  //increase charge until we cover all qBin categories
```

Comment:
This changes the meaning of the bins that use `k` as an index.
Does this affect anything outside of this function ?

Reply:
Yes. For some of the dets, the errors from DB were shifted by one qBin. This correction should affect the position estimate error for clusters associated with those dets. I checked the effect on 500 ttbar events [here](https://mrogulji.web.cern.ch/mrogulji/plots_validate_ibc/index.html) ("harvest_central" vs "harvest_matej") and small differences are present.

Reply:
So this is a bug fix, even when the IBC are not used ?

Reply:
Yes. Do you want me to split this PR into two or correct the description?

Reply:
No need to split the PR, but please update the description mentioning the bug fix.

Reply:
Also, at least the bug fix part should be backported to CMSSW 14.0.x .